### PR TITLE
Fix updating membership expiration date and update UI

### PR DESCRIPTION
**Pull request by davidtanline**
Shows membership expiration date in dropdown menu when editing user's membership expiration as admin.

updatedMembershipExpDate

### CODE REVIEW

**Review by ShaivPandya — APPROVED** [review]

**Review by ShaivPandya — APPROVED** [review]

**Comment by davidtanline** [comment]
Screen Shot 2022-07-04 at 3 59 50 PM
Screen Shot 2022-07-04 at 4 06 18 PM

New AdminDashboard landing page UI and modified navbar links

**Review by neelp03 — APPROVED** [review]

**Review by neelp03 — CHANGES_REQUESTED** [review]
You have some merge conflicts. Also, for the functions you wrote can you add documentation in JS syntax please. Use these for reference: 
- [Inline JS Documentation Guide](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/)
- [Documentation example in Core-v4](https://github.com/SCE-Development/Core-v4/blob/dev/api/peripheral_api/util/RFID-helpers.js#L10:L15)

**Comment by neelp03** [comment]
Also put writeToSQS in a new folder called tools

**Review by evanugarte — APPROVED** [review]

File: api/main_endpoints/util/registerUser.js [hunk 1]
```diff
@@ -29,8 +29,7 @@ function testPasswordStrength(password) {
   return { success: mediumRegex.test(password), message: mediumMessage };
 }
 
-function getMemberExpirationDate(numberOfSemestersToSignUpFor = 0) {
-  /**
+/**
```
Comment (neelp03) [inline api/main_endpoints/util/registerUser.js:32]: add documentation for this too using the js documentation guide
you can also document parameters using @params ... 
all in the guide

**Review by neelp03 — CHANGES_REQUESTED** [review]

**Review by neelp03 — APPROVED** [review]

**Pull request merged**